plugins/event_source/aws_cloudtrail.py

```diff
@@ -0,0 +1,117 @@
+"""
+aws_cloudtrail.py
+
+An ansible-rulebook event source module for getting events from an AWS CloudTrail
+
+Arguments:
+    connection - Parameters use to create AWS session
+    lookup_attributes - List of attributes to filter the events returned. each item of the list should contain an attribute key and value.
+    event_category - Specifies the event category.
+    delay - The number of seconds to wait between polling (default 10sec)
+
+Example:
+
+    - ansible.eda.aws_cloudtrail:
+        connection:
+            aws_access_key_id: access123456789
+            aws_secret_access_key: secret123456789
+            region_name: us-east-1
+            profile_name: default
+        lookup_attributes:
+            - key: eventSource
+              value: ec2.amazonaws.com
```

Comment:
What do you think about having `lookup_attributes` take a mapping of key:value pairs and convert them in the plugin to the format that boto requires?  Similar to `amazon.aws.module_utils.transofrmation.ansible_dict_to_boto3_filter_list` or  `amazon.aws.module_utils.tagging.ansible_dict_to_boto3_tag_list`, like this:

```
        lookup_attributes:
            eventSource: ec2.amazonaws.com
```

Reply:
Good idea, I will do that